libs/features/mas/src/price/index.js

```diff
@@ -15,6 +15,9 @@ const priceOptical = createPriceTemplate({
 const priceStrikethrough = createPriceTemplate({
     displayStrikethrough: true,
 });
+const priceAlternative = createPriceTemplate({
```

Comment:
instead of creating `priceAlternative`, can we

```
const priceStrikethrough = createPriceTemplate({
    displayStrikethrough: true,
    alternativePrice: true|false
});
```

`alternativePrice` could be an option set in inline-price.js based on some attributes or offer data.

Also, likely we need to add this to some of other price rendering function, maybe `pricePromoWithAnnual` ?

happy to discuss this.

Reply:
This is actually a left over from my initial code that should have removed.
However if we want to put the logic into the template.js according to your comment above, we may need this.

We can definitely discuss but I already went through the inline-price's attributes and they don't seem to be a good flag to cover all use cases.
I am not sure how to interpret offer data.